core_lib/src/interface/timelinecells.cpp

```diff
@@ -1068,9 +1129,22 @@ void TimeLineCells::mouseDoubleClickEvent(QMouseEvent* event)
 
     // -- layer --
     Layer* layer = mEditor->object()->getLayer(layerNumber);
-    if (layer && mType == TIMELINE_CELL_TYPE::Layers && event->pos().x() >= 15)
+    if (layer && event->buttons() & Qt::LeftButton)
     {
-        editLayerProperties(layer);
+        if (mType == TIMELINE_CELL_TYPE::Tracks && (layerNumber != -1) && (frameNumber > 0) && layerNumber < mEditor->object()->getLayerCount())
+        {
+            layer->setFrameSelected(frameNumber, false);
```

Comment:
Hmm honestly I don't recall why I did that but I would suspect it to be something with the state not being correct at the given time... maybe caused by another bug.

I will double check to make sure it's needed and otherwise remove it.